src/components/StartPage.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component, Fragment } from 'react';
+import SearchHeader from './SearchHeader';
+import NotFound from './NotFound';
+import styled from 'styled-components'
+import { connect } from 'react-redux';
+import CardFilm from './CardFilm';
+
+const StyledSection = styled.section`
+    padding: 25px;
+    background-color:  rgb(209, 189, 189);
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+
+    div {
+        margin-bottom: 10px;
+    }
+`;
+
+class StartPage extends Component {
+    constructor(props) {
+        super(props)
+    }
+
+    render() {
```

Comment:
The component, which doesn't have neither `state` nor `lifecycle methods` could be written as function component.
It isn't an error, but there are couple benefits of functional components, you could read [here](https://medium.com/@Zwenza/functional-vs-class-components-in-react-231e3fbd7108#c84a)

P.S.: In recent React.js versions there was a warning message if you used a class component without  `state` and `lifecycle methods` , and a suggestion to use the functional one )

Reply:
Tried to rewrite the component, it doesn't work for this case